gm4_better_crossbows/data/better_crossbows/functions/place_torch.mcfunction

```diff
@@ -0,0 +1,19 @@
+# @s = @e[type=minecraft:arrow,tag=gm4_cb_torch,nbt={inGround:1b}]
+# at @s
+
+#place torch
+execute if block ~ ~-.05 ~ #better_crossbows:torch_placeable run setblock ~ ~ ~ minecraft:torch

```

Comment:
Move this line to line 10 to make the "default" standing.

Reply:
it's checking for 0.05 blocks below or to the side, so "default" is only when it's exactly in the at the edge of the block